src/include/miopen/convolution.hpp

```diff
@@ -92,6 +93,25 @@ struct ConvolutionAttribute
         inline bool GetWrW() const { return Get() != 0; } // true is the default.
     } gfx90aFp16alt;
 
+    class deterministic
+    {
+        int value = -1;
+        friend struct ConvolutionAttribute;
+
+        public:
+        inline bool Get() const
+        {
+            int determ_value = value;
+            if(nullptr != miopen::GetStringEnv(MIOPEN_DEBUG_CONVOLUTION_ATTRIB_DETERMINISTIC{}))
+                determ_value = miopen::Value(MIOPEN_DEBUG_CONVOLUTION_ATTRIB_DETERMINISTIC{});
+#if MIOPEN_NDEBUG
+            return determ_value == 1; // false is the default.
+#else
+            return determ_value != 0; // true is the default.
+#endif
```

Comment:
Why this is needed?

Reply:
I just follow this [comments](https://ontrack-internal.amd.com/browse/SWDEV-316705?focusedCommentId=8334110&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-8334110), so I give a default value in debug build or not

Reply:
That comment is unclear. Clarification [requested](https://ontrack-internal.amd.com/browse/SWDEV-316705?focusedCommentId=8378275&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-8378275).